challenge.md

```diff
@@ -6,40 +6,27 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+git clone (repo SSH url)
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+git log
```

Comment:
Nope - `git log` will show our history of commits. We'd probably want to use a combination of `git status` (to show which files have been changed) and  `git diff` (to show all un-added changes).